scripts/monitoring/cron-send-zabbix-too-old.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+
+'''
+This script will download the yum repository metadata for the
+Zabbix 3.0 (LTS) branch and determine the latest version of the
+zabbix-server-mysql package. It will then find version of the
+same RPM as deployed in the environment. It will determine
+how many versions old the running version is and send that
+value to Zabbix.
+'''
+
+#pylint: disable=invalid-name
+
+from __future__ import print_function
+import argparse
+import urllib
+import gzip
+import io
+import os
+import rpm
+import lxml.etree
+import lxml.objectify
+import subprocess
+#pylint: disable=import-error
+from openshift_tools.monitoring.metric_sender import MetricSender
+#pylint: enable=import-error
```

Comment:
It's not needed, but I like re-enabling disabled checks so that the linter can find any other potential issues that do happen to be valid.